nominees/safecast.json

```diff
@@ -0,0 +1,87 @@
+    },
+    {
+      "SDGNumber": 4,
+      "evidenceText": "Safecast’s educational outreach programs and partnerships provide hands-on learning for primary and secondary students to gain skills in engineering and environmental topics that will support future job opportunities and entrepreneurship. Our open and free curriculum ensures that educators have access to affordable and relevant resources.",
+      "evidenceURL": "https://safecast.org/sdg/"
+    },
+    {
+      "SDGNumber": 5,
+      "evidenceText": "Safecast’s global volunteer team is reflective of the demographics they serve. We are committed to encouraging and fostering a diverse and inclusive community.",
+      "evidenceURL": "https://safecast.org/sdg/"
+    },
+    {
+      "SDGNumber": 9,
+      "evidenceText": "Open platforms build resiliency in both communities that lack government support and infrastructure and those with a high GDP. As leadership and budgets change, open, transparent, community-driven innovation, particularly with regards to measuring the environment and climate change, is essential to the continuity and longevity of a critical system. Safecast’s unbiased open platform is adopted by research institutions around the world and the data is endorsed as reliable and accurate.",
+      "evidenceURL": "https://safecast.org/sdg/"
+    },
+    {
+      "SDGNumber": 11,
+      "evidenceText": "Safecast’s openly licensed data platform provides community stakeholders in towns and cities around the world full access to information that they can use to make informed lifestyle choices and influence local policy and legislation.",
+      "evidenceURL": "https://safecast.org/sdg/"
+    },
+    {
+      "SDGNumber": 13,
+      "evidenceText": "Air Quality and particulate matter have been identified as significant health factors by the World Health Organization. Safecast is building a network of sensors to monitor PM1.0, PM2.5, PM 10 that is available for anyone to use for free with no licensing restrictions.",
+      "evidenceURL": "https://safecast.org/sdg/"
+    },
+    {
+      "SDGNumber": 15,
+      "evidenceText": "Safecast is committed to helping people better understand their environment which is interconnected with the ecosystems we share with other living organisms and nonliving objects.",
+      "evidenceURL": "https://safecast.org/sdg/"
+    },
+    {
+      "SDGNumber": 17,
+      "evidenceText": "As an open project, Safecast proudly works with groups across the social and political spectrum and models how to globally scale the dissemination and sharing of information, knowledge, expertise, and technology through open standards.",
+      "evidenceURL": "https://safecast.org/sdg/"
+    }
+  ],
+  "sectors": [
+    "Big Data",
+    "Climate",
+    "Civil society",
+    "Data Collection",
+    "Education",
+    "Health"
+  ],
+  "type": [
+    "software",
+    "data"
+  ],
+  "repositoryURL": "https://github.com/Safecast",
```

Comment:
@Rolikasi could you point to an explicit repo within the [Safecast Github repo](https://github.com/Safecast). Thanks.

Reply:
There are so many repos , but I think that [this one](https://github.com/Safecast/safecastapi) is the most relevant.